Source/Bot/Bot.swift

```diff
@@ -189,11 +190,10 @@ protocol Bot: AnyObject {
     
     // everyone's posts
     func everyone(completion: @escaping PaginatedCompletion)
-    func keyAtEveryoneTop(queue: DispatchQueue, completion: @escaping (MessageIdentifier?) -> Void)
     
     // your feed
     func recent(completion: @escaping PaginatedCompletion)
-    func keyAtRecentTop(queue: DispatchQueue, completion: @escaping (MessageIdentifier?) -> Void)
+    func numberOfRecentItems(since message: MessageIdentifier, completion: @escaping CountCompletion)
```

Comment:
Could you add a doc comment for this function?

Reply:
Sure!